configs/trainers/base.yaml

```diff
@@ -25,8 +25,13 @@ GRAD_CLIP:
   USE: True
   VALUE: 1.0
 DEVICE: "0"
-DISTRIBUTED: False
-DISTRIBUTED_BACKEND: "nccl"
+DISTRIBUTED:
```

Comment:
let's add these config options for command line arguments in `ezflow/tools/train`

Reply:
So there are such 6 such config parameters for distributed training. Do you think we should have all of them as command line arguments? Or we can just have a single flag for enabling distributed training and point the user to the config file for the other parameters?